### PR TITLE
No audio option added - issue #2602

### PR DESCRIPTION
**Pull request by melloskitten**
I added a new option as suggested in the issue.

### CODE REVIEW

File: horizons/main.py [hunk 1]
```diff
@@ -154,6 +154,10 @@ def start(_command_line_arguments):
 	if not setup_gui_logger(command_line_arguments):
 		return False
 
+	## Check if the no-audio flag has been set.
```
Comment (squiddy) [inline horizons/main.py]: These should be single `#`, just like in line 161 and all the others.

**Comment by AndyMender** [comment]
I fixed the comment as @squiddy suggested.

**Comment by squiddy** [comment]
IIRC the previous contribution introduced these as well, so there are probably more places.

**Comment by melloskitten** [comment]
@squiddy I'm sorry! It wasn't my intention. If it helps you guys, this is the other PR I had where I have three comments with ## instead of # https://github.com/unknown-horizons/unknown-horizons/pull/2644/files

**Comment by LinuxDonald** [comment]
fixed.

**Comment by squiddy** [comment]
> @squiddy I'm sorry! It wasn't my intention.

No problem, unfortunately I saw it too late. I don't think our code style checker is configured to warn about that either. :)

**Pull request merged**